### PR TITLE
Added Alerts

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added Info , Success , Error and Warning Alerts

### CODE REVIEW

**Comment by hussiantabha** [comment]
Looks clean & good Atharva

**Comment by ultronjs** [comment]
Looks good

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.alert-success{
+    color:var(--dark-green) ;
+    border: 1px solid var(--medium-green)  ;
+    background-color: var(--light-green) ;
+    
+}
+.alert-error{
+    color: var(--dark-red);
+    border: 1px solid var(--medium-red);
+    background-color: var(--light-red);
```
Comment (barbaricankit) [inline components/alert/alert.css:22]: the only difference here is color and background color, you could have the border property in the alert itself.

File: components/components.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    --medium-green:#75ee75;
+    --light-green:#ddfddd;
+    /* Red */
+    --dark-red:#ff3333;
+    --medium-red:#f34e4e;
+    --light-red:#ffe6e6;
+    /* Orange */
+    --dark-orange:#ffa500;
+    --medium-orange:#ffc04d;
+    --light-orange:#fff6e6;
```
Comment (barbaricankit) [inline components/components.css:19]: Really liked this.

**Review by barbaricankit — COMMENTED** [review]
I can't see the HTML in your code.

File: components/alert/alert.css (hunk 1 above)
Comment (atharvabhanage02) [reply components/alert/alert.css:22]: The border color for each alert is different , so I mentioned it specifically in each alert

File: components/components.css (hunk 2 above)
Comment (atharvabhanage02) [reply components/components.css:19]: > I can't see the HTML in your code.

This is just the CSS file which I'm making for each components and the `components.css` will contain all the CSS files (like for Avatar , Badges , cards ,etc) which the user will use.
I have tested this code in my local machine and just added the CSS file here

**Pull request merged**